examples/github-oidc/main.tf

```diff
@@ -0,0 +1,27 @@
+provider "aws" {
+  region = "eu-west-1"
+}
+
+locals {
+  name        = "role-test"
+  environment = "test"
+}
+
+##----------------------------------------------------------------------------- 
+## GitHub OIDC role module call.   
+##-----------------------------------------------------------------------------
+
+module "aws_github_oidc_role" {
+  source = "../../modules/aws_github_oidc_role"
+
+  enable               = true
```

Comment:
why `enable` attribute here because you are not calling github oidc module in main IAM role module

Reply:
fixed